src/editor/ui_menus/main_menu_map_options.cc

```diff
@@ -508,6 +508,9 @@ MainMenuMapOptions::MainMenuMapOptions(EditorInteractive& parent, Registry& regi
 	ok_.sigclicked.connect([this]() { clicked_ok(); });
 	cancel_.sigclicked.connect([this]() { clicked_cancel(); });
 
+	// This window is accessible from the Save Map fullscreen window, so it needs a higher Z order.
+	set_z(UI::Panel::ZOrder::kFullscreenWindow);
```

Comment:
This is not right when the window is opened from the editor menu or by the keyboard shortcut. Could it be <z_order_of_parent> + 1? But it still needs to work, if the options window is open when the save windoe is opened, then the options button is clicked…